Doc/library/subprocess.rst

```diff
@@ -1149,6 +1149,11 @@ calls these functions.
    code was zero then return, otherwise raise :exc:`CalledProcessError`. The
    :exc:`CalledProcessError` object will have the return code in the
    :attr:`~CalledProcessError.returncode` attribute.
+   .. note::
+
+      If .. function:: `check_call` was run unsuccessfully - it can raise only
```

Comment:
``.. function:: `check_call``` directives are only for blocks of text. Inline markup is slightly different
```suggestion
      If :func:`check_call` was run unsuccessfully - it can raise only
```

Reply:
Thanks for this note! This is fixed in the new version